### PR TITLE
convert pressure hPa to mm Hg

### PR DESCRIPTION
**Pull request by kele6ra**

### CODE REVIEW

**Comment by vharadkou** [comment]
after latest PR, please run first time lint option and change message according the commit lint rules 
example, feat: convert pressure hPa to mm Hg

File: src/app/components/weather/api/dto-converter.ts [hunk 1]
```diff
@@ -1,14 +1,17 @@
 import { WeatherModel } from '../model/weather.model';
 
+
```
Comment (vharadkou) [inline src/app/components/weather/api/dto-converter.ts]: see comment regarding lint

**Comment by kele6ra** [comment]
Updated to last master state

File: src/app/components/weather/api/dto-converter.ts (hunk 1 above)
Comment (kele6ra) [reply src/app/components/weather/api/dto-converter.ts]: Updated to last master state with lint

File: main.ts [hunk 2]
```diff
@@ -36,7 +36,7 @@ function createWindow() {
   }
 
   // for test this option should be disable
-  // win.webContents.openDevTools();
+  win.webContents.openDevTools();
```
Comment (vitalics) [inline main.ts]: maybe setup env variables? if `NODE_ENV=test` than disable devtools?

**Pull request merged**